### ISSUE TITLE
Element.Geometry fails with Value cannot be null, Parameter name: source

### ISSUE THREAD

**Issue by LukeyJohnson** [DynamoDS/Dynamo#5577]
In many cases, it will convert a Room object to a solid without any problem. However, in some cases this error appears. Is this a bug?
![element_geometry](https://cloud.githubusercontent.com/assets/5659896/10627703/4d4effe2-7804-11e5-9c4d-0741d668db34.png)

**Comment by ksobon** [comment]
you have a null value in your list. you are asking it to call Geometry() method on a null value. that will always result in exception unless handled internally.

**Comment by LukeyJohnson** [comment]
The actual list is purely room elements, so it isn't a null value in the list. I'm guess there is something about those room elements that Element.Geometry doesn't like: like maybe they are unbounded, or overlapping, or perhaps volumes are not computed. But I haven't been able to determine what is causing it yet.

**Comment by ksobon** [comment]
I tested this in the Release Candidate 0.9 for Unbounded and Unplaced Rooms and comes through as expected with Empty List for Geometry:

![image](https://cloud.githubusercontent.com/assets/529781/10859101/941372c2-7f30-11e5-9283-ab48687e82f9.png)

Are you on 0.8?

**Comment by LukeyJohnson** [comment]
Yes 0.8.2, probably will be on it for a while.

**Comment by ikeough** [comment]
We're closing this issue as it cannot be reproduced on more recent builds of Dynamo. Please re-open if you're still experiencing this problem.

**Comment by m-Souza** [comment]
Hi everyone. I have the exact same issue while running revit 2016SP2 and Dynamo 1.0.0.1180 on Windows 10.
I have noticed that this is related to room boundaries at points where 2 different wall type join (at least this is what I found while trying to isolate the issue with Room Separation Lines).
The thing is this may be causing the program to crash when I try to "model" a large list of rooms (600+ units).
So here comes my question: Is there a way to identify all the "null rooms" before connecting it to Element.Geometry? 
I tried Object.IsNull on the list but it returns all false while using it after Element.Geometry it shows me which one is causing the error to appear.

Thanks in advance!
